api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,156 @@ module: excel_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to Excel.
 
 ##### return: Promise<CellRange>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells. 
 
 ##### param(options): ExportDataGridProps
-<!-- Description goes here -->
+Exported **DataGrid** settings.
+
+---     
+
+Note that **exportDataGrid(options)** utilizes <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> API to implement export. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+##### jQuery
```

Comment:
There's no need for this code because it duplicates what we show in the demo. A demo link is sufficient.

Reply:
On the second glance, this code will be useful indeed, but then we should clarify why the method is called inside onExporting, why e.cancel, and what the result of this code will be (grid data exported as is).